pkg/cli/admin/upgrade/status/health.go

```diff
@@ -13,6 +13,7 @@ type scopeType string
 const (
 	scopeTypeControlPlane scopeType = "ControlPlane"
 	scopeTypeWorkerPool   scopeType = "WorkerPool"
+	scopeTypeAlert        scopeType = "Alert"
```

Comment:
I don't think we should have a new scope type - I guess assigning alerts to either control plane or worker pool domain is not easy because the alerts do not carry that metadata?

I think that for now we could have a scope type `Cluster` which we could use for items that we cannot pin to a more precise scope.